### PR TITLE
docs: Pass data (desserts) to expansion section

### PR DESCRIPTION
**Pull request by DedicatedManagers**
## Description

Please show how to pass the data down to the expanded section so that one can display extra information that may not fit in the columns.  I just guessed and found "item" to work as the documentation did not help me get there.

I got the template variable wrong it should be info.other, but hopefully you get the idea.

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [ ] The PR title is no longer than 64 characters.
- [ ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by DedicatedManagers** [comment]
@TravisBuddy @jacekkarczmarczyk -- My apologies but I made a mistake in the code.    "item.info" should be "item.other".   How do I resolve this?  Do we drop the merge request and go back and redo the change?  Or can I modify from this point?  (I haven't contributed before... I'm learning and a bit confused. )

File: packages/docs/src/examples/data-tables/intermediate/expand.vue [hunk 1]
```diff
@@ -16,8 +16,8 @@
         <v-switch v-model="singleExpand" label="Single expand" class="mt-2"></v-switch>
       </v-toolbar>
     </template>
-    <template v-slot:expanded-item="{ headers }">
-      <td :colspan="headers.length">Peek-a-boo!</td>
+    <template v-slot:expanded-item="{ headers, item }">
+      <td :colspan="headers.length">Peek-a-boo! {{ item.info }}</td>
```
Comment (DedicatedManagers) [inline packages/docs/src/examples/data-tables/intermediate/expand.vue]: should be `item.other`

Comment (MajesticPotatoe) [reply packages/docs/src/examples/data-tables/intermediate/expand.vue]: need to add `other` to the remainder of the items in `data`. also remove the `Peek-a-boo!`

**Comment by MajesticPotatoe** [comment]
Pulse check

**Comment by MajesticPotatoe** [comment]
Closing due to inactivity and host branch no longer exists.

**Pull request closed without merging**